__tests__/pet.test.js

```diff
@@ -0,0 +1,153 @@
+const Pet = require("../src/pet.js");
+describe("constructor", () => {
+  it("returns an object", () => {
+    expect(new Pet("Fido")).toBeInstanceOf(Object);
+  });
+  it("sets the name property", () => {
+    const pet = new Pet("Fido");
+
+    expect(pet.name).toEqual("Fido");
+  });
+
+  it("has a initial age of 0", () => {
+    const pet = new Pet("Fido");
+
+    expect(pet.age).toEqual(0);
+  });
+  it("has a initial hunger of 0", () => {
+    const pet = new Pet("Fido");
+
+    expect(pet.hunger).toEqual(0);
+  });
+  it("has a initial fitness of 10", () => {
+    const pet = new Pet("Fido");
+
+    expect(pet.fitness).toEqual(10);
+  });
+});
+describe("growUp", () => {
```

Comment:
You should always leave a clear line between each of your test sand describe blocks to break up the text a bit. Would be good if you did this throughout the rest of the file
```
});

describe("growUp", () => {
```